Source/RageCommunity.Library/Pathfinding/VehicleNode.cs

```diff
@@ -41,6 +45,12 @@ public VehicleNode(Vector3 position, Single heading, UInt32 density, Int32 nodeF
         /// </summary>
         public Vector3 Position { get; set; }
 
+        /// <summary>
+        /// The roadside position of the this node.
+        /// </summary>
+        public Vector3 RoadSidePosition { get; private set; }
```

Comment:
Can you tell me an instance where you would want to assign a node with a road side position that doesn't belong to it?

Reply:
For example if you would like to add some self defined nodes, because you have a spot in the game where pathfinding is not working like you need it. 

Another reason for having a setter would be serialization, most of the existing serialization classes in .NET relay on having a SET